homeassistant/components/gtfs/sensor.py

```diff
@@ -619,9 +619,7 @@ def update(self) -> None:
             if not self._departure:
                 self._state = None
             else:
-                self._state = dt_util.as_utc(
-                    self._departure["departure_time"]
-                ).isoformat()
+                self._state = dt_util.as_utc(self._departure["departure_time"])
```

Comment:
UTC conversion is no longer needed.

Reply:
https://github.com/home-assistant/core/pull/61221